src/libgap-api.c

```diff
@@ -42,46 +68,135 @@ Obj GAP_ValueGlobalVariable(const char * name)
     }
 }
 
-//
-// Evaluate a string of GAP commands
-//
-// To see an example of how to use this function
-// see tst/testlibgap/basic.c
-//
-Obj GAP_EvalString(const char * cmd)
+
+////
+//// arithmetic
+////
+
+int GAP_EQ(Obj a, Obj b)
 {
-    Obj instream;
-    Obj res;
-    Obj viewObjFunc, streamFunc;
+    return EQ(a, b);
+}
 
-    streamFunc = GAP_ValueGlobalVariable("InputTextString");
-    viewObjFunc = GAP_ValueGlobalVariable("ViewObj");
+int GAP_LT(Obj a, Obj b)
+{
+    return LT(a, b);
+}
 
-    instream = DoOperation1Args(streamFunc, MakeString(cmd));
-    res = READ_ALL_COMMANDS(instream, False, True, viewObjFunc);
-    return res;
+int GAP_IN(Obj a, Obj b)
+{
+    return IN(a, b);
+}
+
+Obj GAP_SUM(Obj a, Obj b)
+{
+    return SUM(a, b);
+}
+
+Obj GAP_DIFF(Obj a, Obj b)
+{
+    return DIFF(a, b);
+}
+
+Obj GAP_PROD(Obj a, Obj b)
+{
+    return PROD(a, b);
+}
+
+Obj GAP_QUO(Obj a, Obj b)
+{
+    return QUO(a, b);
+}
+
+Obj GAP_LQUO(Obj a, Obj b)
+{
+    return LQUO(a, b);
+}
+
+Obj GAP_POW(Obj a, Obj b)
+{
+    return POW(a, b);
+}
+
+Obj GAP_COMM(Obj a, Obj b)
+{
+    return COMM(a, b);
+}
+
+Obj GAP_MOD(Obj a, Obj b)
+{
+    return MOD(a, b);
+}
+
+
+////
+//// booleans
+////
+
+Obj GAP_True;
+Obj GAP_False;
+Obj GAP_Fail;
+
+
+////
+//// lists
+////
+
+int GAP_IsList(Obj obj)
+{
+    return obj && IS_LIST(obj);
+}
+
+Int GAP_LenList(Obj obj)
+{
+    return LEN_LIST(obj);
+}
+
+void GAP_AssList(Obj list, UInt pos, Obj val)
```

Comment:
Here, I change the names, from e.g. `ASS_LIST` to `AssList`, based on the rule that for other GAP symbols, the `SNAKE_CASE` ones are low-level, while the `CamelCase` ones are ready for use in higher-level code.

Is that a good or a bad idea? You decide -- of course I can also change it to `GAP_ASS_LIST` etc.

Reply:
I have an aesthetic reason to approve the CamelCase (which is probably the reason why `SNAKE_CASE` is used for internal functions, it's really grating)

Reply:
I agree with camelcase as well